test/Specs/LinkedOutputs.purs

```diff
@@ -70,10 +70,35 @@ movingAverages_spec =
    , out_expect: identity
    }
 
+multipleSrcPaths_test :: TestLinkedOutputsSpec
```

Comment:
I don’t think we need a new test, we can just put one of (and eventually all of) the test files in `test/fluid`, which should exercise the new feature.